woocommerce/woocommerce-android
# [UI Tests] Enabled "reviewListShowsAllReviews" test

### What
The test was failing because of the constant refresh of reviews list, once the screen was opened:

https://user-images.githubusercontent.com/73365754/192702827-bea06ad9-a2f7-434a-80d9-67b38ef6b0bb.mov

This blocked from proper scrolling to a certain review and opening it.

### How
My first (wrong) assumption was that the reason is in the usage of Compose elements. The test started failing approximately when Reviews were moved to a `Menu` screen, and this screen is built with Compose. 

However, after checking Logcat, I saw that the notifications mock json (`rest_v11_notifications.json`) is constantly loaded, which made me think that the reason is in outdated mock file. It appeared to be the case. To solve this, I just coped a real response for that request, instead of finding the differences between the mock file and real response. We have no tests for notifications yet, so changes in that mock don't break anything.

- [x] [needed for the fix] Replaced the response in `rest_v11_notifications.json` with a response from current real API
- [x] [refactoring] Removed `notifications_list.json` because it's duplicated by `rest_v11_notifications_list.json`
- [x] [refactoring] Added new key/value pairs to `products_reviews_all.json`

### Testing instructions
- See that `reviewListShowsAllReviews` is green on FTL
- Run the UI tests and a Screenshot test locally - they should pass.
